public/client.html

```diff
@@ -20,11 +20,7 @@
     <meta property="twitter:url" content="http://web.aceattorneyonline.com/">
     <meta property="twitter:image" content="https://repository-images.githubusercontent.com/78860508/89fcba80-aafd-11e9-80db-c5b10c01aba9">
 
-	<meta http-equiv="Content-Security-Policy" content="script-src-elem 'self' 'unsafe-inline' https://ajax.googleapis.com;
-														script-src 'self' 'unsafe-inline' 'unsafe-eval' https://ajax.googleapis.com;
-														style-src-elem 'self' 'unsafe-inline' https://golden-layout.com;
-														style-src 'self' 'unsafe-inline' https://golden-layout.com;
```

Comment:
csp is meant to block xss attacks so we should be diligent about keeping our `script-src` rules accurate. if we don't add any, there will be no protection from XSS

Reply:
I've added them back.